app/javascript/mastodon/utils/scrollbar.ts

```diff
@@ -16,16 +17,3 @@ const getActualScrollbarWidth = () => {
 
   return scrollbarWidth;
 };
-
-export const getScrollbarWidth = () => {
-  if (cachedScrollbarWidth !== null) {
-    return cachedScrollbarWidth;
-  }
```

Comment:
I've removed caching as it would return wrong scrollbar dimensions after a resize.